src/vault_backup/context.py

```diff
@@ -0,0 +1,6 @@
+import sys
```

Comment:
I don't understand the need for this file... you can just import the `backup` module directly into the script file surely.

Reply:
yeah `main` importing `backup` does not rely on `context`. `context` is for importing [modules](https://github.com/artsy/opstools/pull/142/files#diff-b6344671b3e1cbf4db05775c8276bafcabd770323be402b378ca3fc936068d41R9) in `src/lib/`. `context` is the only way i've found to make it work. wonder if there's a better way...

Reply:
I see... if you want to get rid of this file you can prepend `PYTHONPATH=./src` to your `python src/vault_backup/main.py` command. That will fix it.

The other option is to package opstools. You would need to move all code in `src` into an `opstools` sub-folder and issue a `pip install .` command first. Then you can refer to the package as `opstools.lib.logging`. But that may be overkill just to get this script running.

Reply:
ah...thanks for the tips. my feeling is that both options don't seem more attractive than the current approach. i am inclined to leave it as is.